tikv/raftstore/applier.go

```diff
@@ -636,15 +639,22 @@ func (a *applier) handleRaftEntryNormal(aCtx *applyContext, entry *eraftpb.Entry
 	index := entry.Index
 	term := entry.Term
 	if len(entry.Data) > 0 {
-		cmd := new(raft_cmdpb.RaftCmdRequest)
-		err := cmd.Unmarshal(entry.Data)
-		if err != nil {
-			panic(err)
+		var rlog raftlog.RaftLog
+		if entry.Data[0] == raftlog.CustomRaftLogFlag {
```

Comment:
Is it possible that the first byte of `raft_cmdpb.RaftCmdRequest` equals to `raftlog.CustomRaftLogFlag`?

Reply:
👀

Reply:
No, the first byte of the PB is always the same.